packages/atlas/src/components/NotificationTile/NotificationTile.stories.tsx

```diff
@@ -0,0 +1,31 @@
+import { Meta, Story } from '@storybook/react'
+import React, { useState } from 'react'
+
+import { NotificationProps, NotificationTile } from '.'
+
+export default {
+  title: 'Other/NotificationTile',
+  component: NotificationTile,
+  argTypes: {
+    variant: {
+      control: { type: 'radio', options: ['default', 'compact'] },
+    },
+  },
```

Comment:
```suggestion
```
I'm pretty sure that we don't need to add this anymore. This control should be added automatically.